### PR TITLE
Update Powell Teams Connector

### PR DESCRIPTION
**Pull request by PowellSoftware**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Picos** [comment]
Hello,

This validation is in error due to a change in the type of return value, which will lead to a breaking change for our customers.
However, we are aware of this change which is necessary and our customers are and will be informed of this critical breaking change in order to be able to modify their flows.
Currently, the type of this return value does not work, both functionally and technically, and this change therefore had to be made in order to correct these points and thus allow our customers to have a usable and functional connector.

Is it possible to override this validation so that the connector is updated? 

Best regards,
Baptiste Beaupuy-Mouret

**Comment by Amjed-Ayoub** [comment]
@Picos thank you for reaching out. It is very good that you are aware of this change now since we still in GitHub step. You can make this change here. If you make an update later during the certification process, it will reset the process.

**Comment by Amjed-Ayoub** [comment]
Dear partner,

We apologize for the delay you might be experiencing while we are working on a high volume of connector requests. Rest assured that we are doing everything we can to address this volume. We are taking measures like improving our processes and hiring additional team members to guarantee resolving your connector requests in a timely manner. 

Please let us know if you have any questions in the meantime.

File: certified-connectors/Powell Teams/README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ In order to use this API, you need to be authorized. To do so, click on the "Aut
 - Notes.Read.All
 - Notes.ReadWrite.All
 - Sites.Read.All
+- User.Invite.All
+- User.Read
 - User.Read.All
```
Comment (v-eisljami) [inline certified-connectors/Powell Teams/README.md:25]: Please review this [template ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and add the required sections to your Readme.md. All connectors must follow template.

**Comment by Amjed-Ayoub** [comment]
Hi @PowellSoftware, I hope you are doing well. I am following up with you to see how thing are going with updating your connector.
Please let me know if you have any questions.
Thank you very much

**Comment by Amjed-Ayoub** [comment]
Hi @PowellSoftware, I am following up to check if you need any help with updating your connector files.
Please let me know if you have any questions.
Thank you very much.

**Comment by sriyen-msft** [comment]
@PowellSoftware I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Comment by Amjed-Ayoub** [comment]
Hi @PowellSoftware I hope you are doing well. I am following up with you to see how things are going with updating your artifacts.
Please let us know if you have any questions.
We look forward to getting your connector certified.
Thank you very much.

**Comment by Amjed-Ayoub** [comment]
Hello @PowellSoftware,
This is to follow up on the status of your connector. I would like to offer help to you if you are having any issues with your connector.
Please do not hesitate to reach out if you have any questions as we are always happy to help.
Thank you very much for working with us.

**Comment by Amjed-Ayoub** [comment]
Hello @PowellSoftware,
I hope you are doing well.
I am following up with you to see how things are going with preparing your connector.
Please let us know if you have any questions.
Thank you.

**Comment by Amjed-Ayoub** [comment]
Hello Partner,
I hope you are doing well.
I am following up in my messages above to see if you need any help with updating your connector artifacts.
Please do not hesitate to reach out to me if you have any questions as we are always here to help. 😊
I look forward to hearing from you.
Thank you very much.

**Comment by sriyen-msft** [comment]
@PowellSoftware  We are closing your PR since there is no response from your side. You can reopen the same PR whenever you are ready for certification of your connector, thanks.

**Pull request closed without merging**